smarty/templates/main.tpl

```diff
@@ -41,45 +41,14 @@
               ReactDOM.render(breadcrumbs, document.getElementById("breadcrumbs"));
             {/if}
 
-            // If <input type="date" /> is not supported (i.e. Firefox), load
-            // jquery date-picker
-            if (!Modernizr.inputtypes.date) {
-              var dateInputs = $('input[type=date]');
-              dateInputs.datepicker({
-                dateFormat: 'yy-mm-dd',
-                changeMonth: true,
-                changeYear: true,
-                yearRange: "1900:" + new Date().getFullYear(),
-                constrainInput: true
-              });
-              dateInputs.attr('placeholder', 'yyyy-mm-dd');
-              dateInputs.on('keydown paste', function(e) { e.preventDefault(); });
-            }
-
-            if (!Modernizr.inputtypes.month) {
-              var monthInputs = $('input[type=month]');
-              monthInputs.datepicker({
-                dateFormat: 'yy-mm',
-                changeMonth: true,
-                changeYear: true,
-                yearRange: "1900:" + new Date().getFullYear(),
-                constrainInput: true,
-                onChangeMonthYear: function(y, m, d) {
-                  // Update date in the input field
-                  $(this).datepicker('setDate', new Date(y, m - 1, d.selectedDay));
-                }
-              });
-              monthInputs.attr('placeholder', 'yyyy-mm');
-              monthInputs.on('keydown paste', function(e) { e.preventDefault(); });
-            }
-
             // Initialize bootstrap tooltip for site affiliations
             $('#site-affiliations').tooltip({
               html: true,
               container: 'body'
             });
           });
         </script>
+        <script src="{$baseurl}/js/firefoxDates.js" type="text/javascript"></script>
```

Comment:
This should be added to the NDB_Page `getJSDependencies` function instead of here